src/gravel/controllers/services.py

```diff
@@ -146,8 +146,7 @@ def check_requirements(
     def _create_service(self, svc: ServiceModel) -> None:
         if svc.type == ServiceTypeEnum.CEPHFS:
             self._create_cephfs(svc)
-        else:
-            raise NotImplementedError("only cephfs is currently supported")
+        raise NotImplementedError(f"unknown service type: {svc.type}")
```

Comment:
Shouldn't this statement be under the else block?

Reply:
oops! yes ... fixed :)